ui/src/views/user/survey-edit/components/SurveyBuild.vue

```diff
@@ -44,20 +44,23 @@
 import FormBuilder from "@/form-builder/components/FormBuilder.vue";
 export default {
   name: "SurveyBuild",
-  components: {
-    FormBuilder
-  },
-  data() {
-    return {
-      formData: {
-        title: "Survey title",
-        description: "",
-        formBuilder: {
-          list: [],
-          models: {}
+  props: {
+    formData: {
```

Comment:
and i see you moved the formData from data to prop. They are different, we shouldnt modify prop :"  (I did it myself but it should be refactored because its a realllyyyy baddddd praaaccctiiicceeee. Prop should be used to store static data / data fed by parent, while data is reactive.